crates/swc_ecma_lints/src/rules/prefer_const.rs

```diff
@@ -11,15 +12,21 @@ use crate::{
 // todo: implement option destructuring: all | any
 // https://eslint.org/docs/rules/prefer-const#destructuring
 
-// todo: implement option ignoreReadBeforeAssign
-// https://eslint.org/docs/rules/prefer-const#ignorereadbeforeassign
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct PreferConstConfig {
+    ignore_read_before_assign: Option<bool>,
```

Comment:
I think we need `#[serde(default)]`

Reply:
And it can be `bool` instead of `Option<bool>`.

Reply:
Sorry for late fix. I was busy